lib/spark_api/models/message.rb

```diff
@@ -13,11 +14,17 @@ def save(arguments={})
         end
         false
       end
+
       def save!(arguments={})
         results = connection.post self.class.path, {"Messages" => [ attributes ]}, arguments
         true
       end
-      
+
+      def replies(args = {})
+        arguments = {:_expand => "Body, Sender"}.merge(args)
+        @replies ||= Message.collect(connection.get("#{self.class.path}/#{self.Id}/replies", arguments))
```

Comment:
If we call this, then a new reply comes in, and we call this again - will the new reply show up in `@replies`?